.github/workflows/post_release_version_bump.yml

```diff
@@ -0,0 +1,39 @@
+name: Automated Release Version Bumps
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  post_release_version_bump:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+
+      - name: Install Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install pennylane
+
+      - name: Run version bump
+        run: >
+          python .github/workflows/vb_script.py
+          --version_path "./pennylane_cirq/_version.py"
+          --changelog_path "CHANGELOG.md" --post_release --pre_pl_release
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3
+        with:
+          commit-message: post release version bump
+          title: Version Bump
+          body: updated changelog and _version.py
+          branch: post-release-version-bump
+          reviewers: Jaybsoni
```

Comment:
Will change this to whom ever is assigned as reviewer for this plugin

Reply:
@rmoyard would you be okay to be tagged here? Whenever the action is triggered, you'd be assigned as the reviewer for the pre-release/post-release version bump PR.

Reply:
@Jaybsoni @antalszava You can add me as reviewer!

Reply:
```suggestion
          reviewers: rmoyard
```